src/Compilers/VisualBasic/Portable/Syntax/Syntax.xml

```diff
@@ -7466,7 +7466,7 @@ Unify Enum declarations with other block type declarations.
       </child>
     </node-structure>
 
-    <node-kind-alias name="NonArrayTypeName" alias="PredefinedType|NullableType|@NameSyntax">
+    <node-kind-alias name="NonArrayTypeName" alias="PredefinedType|NullableType|@NameSyntax|TupleType">
```

Comment:
>|TupleType [](start = 91, length = 10)

It feels like this is affecting more scenarios than this PR is claiming to address. For example an attribute syntax. I think the fix should be more targeted.